README.md

```diff
@@ -93,6 +93,8 @@ When using the _GITHUB_TOKEN_, the **minimum required permissions** are:
 | `releasedLabels`      | The [labels](https://help.github.com/articles/about-labels) to add to each issue and pull request resolved by the release. Set to `false` to not add any label. See [releasedLabels](#releasedlabels). | `['released<%= nextRelease.channel ? \` on @\${nextRelease.channel}\` : "" %>']-                                                                     |
 | `addReleases`         | Will add release links to the GitHub Release. Can be `false`, `"bottom"` or `"top"`. See [addReleases](#addReleases).                                                                                  | `false`                                                                                                                                              |
 | `draftRelease`        | A boolean indicating if a GitHub Draft Release should be created instead of publishing an actual GitHub Release.                                                                                       | `false`                                                                                                                                              |
+| `releaseName`         | A [Lodash template](https://lodash.com/docs#template) to customize the github release's name                                                                                                           | `<%= nextverison.name %>`                                                                                                                            |
+| `releaseBody`         | A [Lodash template](https://lodash.com/docs#template) to customize the github release's body                                                                                                           | `<%= nextverison.notes %>`                                                                                                                           |
```

Comment:
should we rename the arguments to `releaseNameTemplate`/`releaseBodyTemplate` for clarity?

Reply:
Good call, updated